docs/content/v0.20/architecture.md

```diff
@@ -1,11 +1,12 @@
+**NOTE**: It was decided by the community to rename `Placement` -> `BindingPolicy`.  
+This document uses the new terminology `BindingPolicy` for the resource that was called previously `Placement`, so on every place `BindingPolicy` is mentioned in this doc, we mean `Placement` as it was used up until today.
```

Comment:
This doc also talks about the `Binding` Kind, so its relation to the current truth (`PlacementDecision`) should be noted.